### PR TITLE
Niloofar/FEQ-1692/Enhancing quill-icons Release Workflow with Automated Slack Notifications

### PR DESCRIPTION
**Pull request by niloofar-deriv**
As a developer, I want to implement automated Slack notifications after each release, so that everyone are instantly informed about the latest updates. This will improve communication, facilitate prompt bug fixes, and enhance overall collaboration within the team.

### CODE REVIEW

**Review by jim-deriv — APPROVED** [review]

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -52,3 +57,27 @@ jobs:
...
             -d '{"ref":"main"}'
+
+  send_slack_notification:
+    name: Send Release Slack notification
+    runs-on: ubuntu-latest
+    if: always()
+    needs: [release]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
```
Comment (shaheer-deriv) [inline .github/workflows/release.yaml:68]: `actions/checkout@v4` ?

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -52,3 +57,27 @@ jobs:
...
+      - name: Conclusion
+        uses: technote-space/workflow-conclusion-action@45ce8e0eb155657ab8ccf346ade734257fd196a5
```
Comment (shaheer-deriv) [inline .github/workflows/release.yaml:70]: `technote-space/workflow-conclusion-action@v3.0.3` ?

File: .github/workflows/release.yaml (hunk 1 above)
Comment (jim-deriv) [reply .github/workflows/release.yaml:68]: Hi @shaheer-deriv, this is the way that security team recommended to deal with third party actions. I'll DM you the  guide.

Comment (niloofar-deriv) [reply .github/workflows/release.yaml:68]: Thanks @jim-deriv 🙏

File: .github/workflows/release.yaml (hunk 2 above)
Comment (niloofar-deriv) [reply .github/workflows/release.yaml:70]: As @jim-deriv mentioned this is related to security team's recommendation

**Review by shaheer-deriv — APPROVED** [review]

**Pull request merged**